crates/cli/src/subcommands/build.rs

```diff
@@ -32,7 +32,7 @@ pub fn cli() -> clap::Command {
         )
 }
 
-pub async fn exec(_config: Config, args: &ArgMatches) -> Result<(), anyhow::Error> {
+pub async fn exec(_config: Config, args: &ArgMatches) -> Result<PathBuf, anyhow::Error> {
```

Comment:
This now returns the path of the built binary, in case a caller wants it (similar to how `crate::tasks::build` does)